parser/Builder.cc

```diff
@@ -354,9 +354,18 @@ class Builder::Impl {
             }
         }
 
+        bool numblock = false;
```

Comment:
`numblock` is also the name of a function in this class. Would you mind renaming this to like `needsNumblock`?

Reply:
I went with `isNumblock`